server/models/Category.model.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const categorySchema = new mongoose.Schema({
+  category: {
+    type: String,
+    trim: true,
+    uppercase: true,
+    required: true,
+  },
+  // references the user object that this TODOItem belongs to
```

Comment:
Probably delete or edit this comment since this is in Category

Reply:
good catch. Thanks